File: C & C++ PRACTICE CODES/Nested for Pattern 4/code6.c
#include <stdio.h>

void main (){

	int row;
	printf("Enter no. of rows : ");
	scanf("%d",&row);
	
	int count ;
	
 if(row%2==0){

                count =2;
        }else{
                count = 1;
        }

	

	int temp = 0;
	for(int i = 1; i <= row ; i++){

		
		for(int j = 1; j <= row ; j++){

			
				printf("%d 	",count);
		
			while(temp == 0 ){
				count ++;
				temp = 1;
				for(int  k = 2; k <= count/2  ; k++){

					
					if(count%k == 0){

						temp = 0;
						break;
					}
				}
			}

		
			temp =0;
			
		}
		printf("\n\n");

	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code6.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp = row*row;
		for(int i = 1; i <= row ; i++){
			

			for(int j = 0; j < row ; j++){
					System.out.print((temp)+"		");
				
				temp--;
			}

				System.out.print("\n\n");

		}

	}

}
